lib/nylas/message.rb

```diff
@@ -69,5 +69,26 @@ def expanded
       assign(api.execute(method: :get, path: resource_path, query: { view: "expanded" }))
       self
     end
+    
+    def save_call
+      handle_folder()
+
+      execute(
+        method: :put,
+        payload: attributes.serialize(keys: allowed_keys_for_save),
+        path: resource_path
+      )
+    end
+
+    def handle_folder
+      return if folder.nil?
```

Comment:
silly question but shouldn't we check in `self`?

Reply:
@khamidou Great question!  I thought is was required as well, but its not - https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RedundantSelf.  For example, the linter is saying that `!self.to_h.dig(:folder, :id).nil?` does not need self.  I think having self is probably more readable, so happy to add that if you agree.

Reply:
Not a rubyist but if it works it works.